src/main/java/com/teamcautionrobotics/robot/Cargo.java

```diff
@@ -70,6 +71,7 @@ public void deployExitFlap(boolean goingUp) {
         } else {
             exitFlapDeployer.set(goingUp);
         }
+        currentExitFlapState = goingUp;
```

Comment:
I like this implementation better than the original of updating `currentExitFlapState` in the toggle method.